SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -93,52 +93,67 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 
 	FString ClientWorkerAttribute = GetOwnerWorkerAttribute(Actor);
 
-	WorkerAttributeSet ServerAttribute = { SpatialConstants::ServerWorkerType };
-	WorkerAttributeSet ClientAttribute = { SpatialConstants::ClientWorkerType };
-	WorkerAttributeSet OwningClientAttribute = { ClientWorkerAttribute };
+	WorkerRequirementSet AnyServerRequirementSet;
+	WorkerRequirementSet AnyServerOrClientRequirementSet = { SpatialConstants::UnrealClientAttributeSet };
 
-	WorkerRequirementSet ServersOnly = { ServerAttribute };
-	WorkerRequirementSet ClientsOnly = { ClientAttribute };
-	WorkerRequirementSet OwningClientOnly = { OwningClientAttribute };
+	WorkerAttributeSet OwningClientAttributeSet = { ClientWorkerAttribute };
+
+	WorkerRequirementSet AnyServerOrOwningClientRequirementSet = { OwningClientAttributeSet };
+	WorkerRequirementSet OwningClientOnlyRequirementSet = { OwningClientAttributeSet };
+
+	const TArray<FName>& ServerWorkerTypes = GetDefault<USpatialGDKSettings>()->WorkerTypes.Array();
```

Comment:
Just for encapsulating the information in one place so GDK doesn't have to look up settings or some other reason?

Reply:
Seems like SpatialGDKSettings are grabbed from all over the GDK currently so I'm not sure the benefit will be properly realised / in future work people may just end up grabbing them from settings too and it may be better just to have that as the general source of truth?

Reply:
For WorkerTypes this makes sense actually, since they're not tied to actor groups atm. Ignore me 😃

Reply:
I'm not sure, if we do then ActorGroupManager is no longer an implementation detail of the ClassInfo manager. Is there any other usecase for having direct access to the ActorGroupManager?

Reply:
USpatialStatics in https://github.com/spatialos/UnrealGDK/pull/1063 are a use case for directly accessing ActorGroupManager elsewhere in GDK code 👍

Reply:
But that's not to say the ClassInfoManager could actually own the pointer to the ActorGroupManager since the SpatialNetDriver does nothing but initialise it and pass the reference to the ClassInfoManager (I've raised this on the actor groups PR).